packages/rocketchat-livechat/.app/client/views/message.html

```diff
@@ -10,7 +10,13 @@
 				{{/if}}
 			</span>
 			<div class="body" dir="auto">
-				{{{body}}}
+				{{#if hasAttachments}}
+					{{#each attachments}}
+						{{injectIndex . @index}} {{> messageAttachment}}
```

Comment:
is this `index` from `injectIndex` being used somewhere? or what is its purpose?

Reply:
My bad.. I was probably thinking of something like ReactJS approach.
Fixed.